Tests/scripts/infrastructure_tests/common_test.py

```diff
@@ -1,147 +1,174 @@
-from Tests.scripts import common
+from Tests.scripts.common import get_reviewer, get_person_in_charge, are_pipelines_in_order, is_pivot, get_slack_user_name
+import pytest
+from requests_mock import MockerCore
 
 
-def test_person_in_charge(mocker):
+def test_get_person_in_charge(mocker):
     """
     Given a commit object
-    When person_in_charge is called on it
-    Then it should return the expected name, email and PR link
+    When get_person_in_charge is called on it
+    Then it should return the expected name and PR link
     """
     commit = mocker.Mock()
     commit.author_name = 'John Doe'
     commit.author_email = 'john@doe.com'
     commit.title = 'Merge branch \'master\' into branch-name (#123)'
 
-    expected = ('John Doe', 'john@doe.com', 'https://github.com/demisto/content/pull/123')
+    expected = ('John Doe', 'https://github.com/demisto/content/pull/123')
 
-    result = common.person_in_charge(commit)
+    result = get_person_in_charge(commit)
 
     assert result == expected
 
 
-def test_are_pipelines_in_order_as_commits_true(mocker):
+@pytest.mark.parametrize(('pipeline1_date, pipeline2_date, expected'), (
+    pytest.param('2020-01-01T00:00:00Z', '2020-01-02T00:00:00Z', False, id="pipelines not in order"),
+    pytest.param('2020-01-02T00:00:00Z', '2020-01-01T00:00:00Z', True, id="pipelines in order")))
+def test_are_pipelines_in_order(mocker, pipeline1_date, pipeline2_date, expected):
```

Comment:
```suggestion
def test_pipelines_are_in_correct_order(mocker):
...
   old_pipeline = ...
   new_pipeline = ...
   assert are_pipelines_in_order(old_pipeline, new_pipeline) == True
   
 def test_pipelines_are_not_in_correct_order():
 ...
```
don't use parameterize